src/wrapper-start.js

```diff
@@ -37,18 +37,11 @@
     var newErr;
     if (err instanceof Error) {
       var newErr = new Error(err.message, err.fileName, err.lineNumber);
-      if (isBrowser) {
-        newErr.message = err.message + '\n\t' + msg;
-        newErr.stack = err.stack;
-      }
-      else {
-        // node errors only look correct with the stack modified
-        newErr.message = err.message;
-        newErr.stack = err.stack + '\n\t' + msg;
-      }
+      newErr.message = msg + '\n\t' + err.message;
```

Comment:
I'm sure about reversing this ordering - the whole point is that the error starts with the source, and goes outwards through the calling sites. Not the reverse.